dali/pipeline/pipeline.h

```diff
@@ -96,7 +96,7 @@ class DLL_PUBLIC Pipeline {
   }
 
   DLL_PUBLIC Pipeline(const string &serialized_pipe, int batch_size = -1, int num_threads = -1,
-                      int device_id = -1, bool pipelined_execution = true,
+                      int device_id = -1, int64_t seed = -1, bool pipelined_execution = true,
```

Comment:
Now any code that uses the old `Pipeline(const string &serialized_pipe`  signature is brokenand no one knows why.

Reply:
How about moving it to the last argument?

Reply:
True, changed

Reply:
Now it is inconsistent in a different way with other constructors.